services/ui-src/src/verbiage/pages/mcpar/mcpar-intro.ts

```diff
@@ -1,6 +1,51 @@
+import NavigationSectionsImage from "../../../assets/images/NavigationSections_2x.png";
+import NavigationSectionsSubmissionImage from "../../../assets/images/NavigationSectionsSubmission_2x.png";
+
```

Comment:
Eventually all of these will be stored as JSON files, so rather than storing the images here, let's move them to be imported inside the intro page and called directly in the image sections 2 and 3. That way we're not referencing dynamic things in what should just be a static file.